InfiniteScrollView.js

```diff
@@ -70,10 +89,22 @@ class InfiniteScrollView {
     }
 
     if (this._distanceFromEnd(event) < this.props.distanceToLoadMore) {
-      this.props.onLoadMore();
+      this._onLoadMore();
     }
   }
 
+  _onLoadMore() {
```

Comment:
seems useful and not too hard to maintain for this to be `_onLoadMoreAsync` and be an async function that uses await and try-catch.